packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -102,6 +118,19 @@ export const SocialShare = ({
         onClick={() => trackClick(ShareProvider.Email)}
         label="Email"
       />
+      {!isComment &&
+        !post.private &&
+        squads.map((squad) => (
+          <button
+            type="button"
+            className="flex flex-col items-center"
+            key={squad.id}
+            onClick={() => onShareToSquad(squad)}
+          >
+            <SourceProfilePicture source={squad} />
+            <ShareText className="mt-2">@{squad.handle}</ShareText>
```

Comment:
Shouldnt we use the name?

Reply:
Names are often so much longer and can be the same with other Squads. I think going with the handle has its benefits. WDYT?